bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/util/LabelImpl.java

```diff
@@ -31,12 +31,10 @@ public class LabelImpl implements Label {
     private static final String PN_IS_TITLE_RICH_TEXT = "isTitleRichText";
 
     private ValueMap properties;
-    private String defaultTitle;
     private I18n i18n;
 
-    public LabelImpl(Resource field, String defaultTitle, I18n i18n) {
```

Comment:
What will be the default title if you remove this ?

Reply:
Here we were using getName() method as default title. Since it was not required as we are checking if jcr:title is present i have removed the reference.